### ISSUE TITLE
Upgrade issues

### ISSUE THREAD

**Issue by jdnichollsc** [Microsoft/AppCenter-SDK-React-Native#226]
Hi guys,

I was using `appcenter appcenter-analytics appcenter-crashes` with the version 1.0.1. After follow the configurations to enable Code Push, my packages were updated to `1.1.0`. I removed my `node_modules` folder and I executed the following commands:
```
npm cache clean
npm i
react-native-git-upgrade
react-native link
react-native start
```
But I'm getting the following issue
<img width="1106" alt="captura de pantalla 2018-01-24 a la s 3 42 48 p m" src="https://user-images.githubusercontent.com/2154886/35356727-3ed67822-011f-11e8-8e21-6142280a8a6e.png">

Let me know what you think, thanks in advance!

**Comment by dhei** [comment]
Hi @jdnichollsc, 

The error on the red screen suggests there is ambiguous resolution of two react-native folders. It is likely to be `react-native-git-upgrade` that caused the problem. Have you followed the error suggestion to delete the extra `package.json` to fix this issue? Did it work?

If you only want to upgrade appcenter package version, then you don't need `react-native-git-upgrade` as it will upgrade your react-native version as well as other npm dependencies. To upgrade appcenter version, simply `npm install appcenter@1.1.0 --save && react-native link` should work.

PS: I personally have seen quite a few issues with `react-native-git-upgrade` that messed up npm dependencies in the past, but I have never digged deep enough to understand the root cause. You might want to check out [this GitHub issue on React Native about ambiguous resolution](https://github.com/facebook/react-native/issues/15789) to see if it helps to solve any problem.

**Comment by jdnichollsc** [comment]
@dhei Thanks for your quickly response! After install the packages and execute `react-native link`, I'm getting the following error from the terminal:

```
Scanning folders for symlinks in /Users/.../reactnative/react-native/node_modules (21ms)
Configuring AppCenter
Android App Secret is '...' set in android/app/src/main/assets/appcenter-config.json
rnpm-install info Android module appcenter is already linked 
rnpm-install info iOS module appcenter is already linked 
iOS App Secret is '...' set in ios/myApp/AppCenter-Config.plist
Added code to initialize iOS AppCenter SDK in ios/myApp/AppDelegate.m
Analyzing dependencies
Fetching podspec for `RNShare` from `../node_modules/react-native-share`
[!] CocoaPods could not find compatible versions for pod "AppCenter/Core":
  In snapshot (Podfile.lock):
    AppCenter/Core (= 1.0.1)

  In Podfile:
    AppCenter/Crashes (~> 1.0.1) was resolved to 1.0.1, which depends on
      AppCenter/Core (= 1.0.1)

    AppCenterReactNativeShared (~> 1.1.0) was resolved to 1.1.0, which depends on
      AppCenter/Core (= 1.1.0)

Specs satisfying the `AppCenter/Core (= 1.0.1), AppCenter/Core (= 1.1.0)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by jdnichollsc** [comment]
Ohh I removed all references of AppCenter and then I could install the new version without problems! 👍
